### DISCUSSION TITLE
Make environment variables more easily accessible in CP

### DISCUSSION THREAD

**Discussion by carlcs** [craftcms/cms#9112] [Ideas]
It’s not especially easy to get to a list of available environment variables in the CP. Scrolling to the Environment section in the PHP Info utility can be frustrating, given that the page is so long.

Maybe also listing them in System Report could be a way to make them more accessible. Or adding a new utility Environment, that lists environment variables and Yii aliases (https://github.com/craftcms/cms/issues/6992).

**Comment by carlcs** [comment]
Another issue with the list in PHP info is that it isn’t in alphabetical order.

**Comment by brandonkelly** [comment]
I thought about adding this when adding the list of aliases, but there are typically _a lot_ of environment variables, so I’m a little worried that it would be overwhelming.

**Comment by carlcs** [comment]
FYI to anyone following this issue, I just noticed that there’s now a plugin that lists environment variables in a utility.

> This plugin also provides a handy utilities page where you can view key environment variables useful during development.  
[Hop-reveal](https://plugins.craftcms.com/hop-reveal)
